docs/guide/api-specs.md

```diff
@@ -109,7 +109,7 @@ This helps users understand:
   "paymasterAddress": "0x...",
   "feeQuotes": [
     {
-      "tokenAddress": "0x...",
+      "token": "0x...",
```

Comment:
@web3nish @shunsukew please comfirm it's the same case for quotes api as well.

Reply:
we don't publish quotes api I guess. @web3nish